### ISSUE TITLE
[v3.1.8] Calendar shows events one day ago, even if it is deactivated.

### ISSUE THREAD

**Issue by CptWindows** [Zefau/ioBroker.jarvis#2566]
Hi, 
the calendar shows events one day ago, although the value is set to 0 in the configuration.
This probably occurs because it takes the last 24 hours and the event was an all-day event, right?

Jarvis Version: v3.1.8
![grafik](https://github.com/Zefau/ioBroker.jarvis/assets/13573595/93411eba-dfa4-462c-b85a-2702817724b6)
![grafik](https://github.com/Zefau/ioBroker.jarvis/assets/13573595/4a02bf36-1872-41c5-9be2-61979a678c39)

**Comment by Zefau** [comment]
No, the problem is not that it is an all-day event and jarvis is not looking back 24hours, only until 00:00.

The problem is, that the start time is "01:00" for some reason. Can you provide the raw ical data for the event? Do you know which timezone is set for the event?
